elektra/parallel_euler_tour_tree/hdt_element.h

```diff
@@ -74,11 +74,11 @@ class HdtElement : public ElementBase<HdtElement, _internal::HdtAugmentation> {
   // to be pushed down to the next level).
   parlay::sequence<std::pair<v_int, v_int>> GetAndClearLevelIEdges();
 
-  // For each HdtElement elements[i] (which should represent a vertex), updates
+  // For each HdtElement elements[i] (which should represent a vertex), increments
   // its count of "number of level-i non-tree edges incident to this element" to
   // be new_values[i].
   template <typename ElemSeq, typename IntSeq>
-  static void UpdateNontreeEdgeCounts(const ElemSeq& elements, const IntSeq& new_values);
+  static void IncrementNontreeEdgeCounts(const ElemSeq& elements, const IntSeq& increment);
```

Comment:
It's easier for `dynamic_connectivity.h` to determine how much to increment the counts by as opposed to specifying the counts themselves